client/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
I think that this is not your apps name.

Reply:
Ok will change that